.github/workflows/docs.yml

```diff
@@ -0,0 +1,25 @@
+name: Deploy Sphinx documentation to Pages
+
+on:
+  push:
+    branches: ["main"]
```

Comment:
(optional) we could start having a release branch that points to current release on pip / conda. that way docs match the thing people download instead of dev version in the repo?

Reply:
good point 👍, could be done together with the JOSS-paper release/ as soon as the changes from #31 are merged